src/helpers/buildTransactionsSectionsSelector.tsx

```diff
@@ -119,18 +128,14 @@ export const buildTransactionsSections = ({
     }
   }
 
-  // i18n
-  let requestsToApprove: any = [];
   if (!isEmpty(requests)) {
-    requestsToApprove = [
-      {
-        data: requests,
-        renderItem: ({ item }: any) => <RequestCoinRow item={item} theme={theme} />,
-        title: i18n.t(i18n.l.walletconnect.requests),
-      },
-    ];
+    sectionedTransactions.push({
+      data: requests,
+      renderItem: ({ item }: { item: WalletconnectRequestData }) => <RequestCoinRow item={item} theme={theme} />,
+      title: i18n.t(i18n.l.walletconnect.requests),
+    });
   }
   return {
-    sections: requestsToApprove.concat(sectionedTransactions),
+    sections: sectionedTransactions as SectionListData<TransactionItemForSectionList, TransactionSections>[],
```

Comment:
I think this is changing the order, before wc requests would be at the start of the list, now I think they are in the end

Reply:
good catch. I'll `unshift` it